src/components/settings/SpendingLimits/SpendingLimitsTable.tsx

```diff
@@ -88,9 +85,9 @@ export const SpendingLimitsTable = ({
       { id: 'beneficiary', label: 'Beneficiary' },
       { id: 'spent', label: 'Spent' },
       { id: 'resetTime', label: 'Reset time' },
-      { id: 'actions', label: 'Actions', sticky: true, hide: shouldHideactions },
```

Comment:
`hide` can now be removed from `EnhancedCell['hide']` as well as the associated CSS.